.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+  - "2.7"
+install:
+  - pip install zc.buildout
+  - buildout
```

Comment:
It's a fair bit to ask, but I'd love to lose the dependency on buildout.
Here's an example of the kind of .travis.yml I'd love to see, and indeed, the project structure I hope we can get to:

https://github.com/Simplistix/mush/blob/master/.travis.yml